ci-operator/step-registry/openstack-k8s-operators/build/openstack-k8s-operators-build-commands.sh

```diff
@@ -126,6 +126,12 @@ if [[ "$BASE_OP" != "$META_OPERATOR" ]]; then
   else
     API_SHA=${PR_SHA}
     REPO_NAME=${PR_REPO_NAME}
+    # NOTE(dviroel): We need to replace registry in bundle only when testing
+    #  a PR against operator's repo. When testing rehearsal jobs, we consume
+    #  from latest commit.
+    export IMAGENAMESPACE=${ORGANIZATION}
+    export IMAGEREGISTRY=${REGISTRY}
+    export IMAGEBASE=${SERVICE_NAME}
   fi
 
   go mod edit -replace github.com/${ORG}/${BASE_OP}/api=github.com/${REPO_NAME}/api@${API_SHA}
```

Comment:
Another optimization is to add a condition to only run the `go mod` edit for IS_REHEARSAL=false case.